### PR TITLE
add conformance tag to fast parallel registry tests

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: test/extended/imageapis/limitrange_admission.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 
 const limitRangeName = "limits"
 
-var _ = g.Describe("[Feature:ImageQuota][registry][Serial] Image limit range", func() {
+var _ = g.Describe("[Feature:ImageQuota][registry][Serial][Suite:openshift/registry/serial] Image limit range", func() {
```
Comment (smarterclayton) [inline test/extended/imageapis/limitrange_admission.go]: At the end, please

File: test/extended/images/resolve.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[Feature:ImageLookup][registry] Image policy", func() {
+var _ = g.Describe("[Feature:ImageLookup][registry][conformance] Image policy", func() {
```
Comment (smarterclayton) [inline test/extended/images/resolve.go]: Conformance

File: test/extended/imageapis/limitrange_admission.go (hunk 1 above)
Comment (bparees) [reply test/extended/imageapis/limitrange_admission.go]: ginkgo concatenates this text w/ the additional test text, so i think it makes more sense to put the tags at the beginning so you don't get a weird concatenation with the tag in the middle.  What's the argument for putting it at the end?

Comment (smarterclayton) [reply test/extended/imageapis/limitrange_admission.go]: that's where the automatic suite goes

Comment (bparees) [reply test/extended/imageapis/limitrange_admission.go]: well that seems like a bad idea too.  why double down on it? :)

**Comment by bparees** [comment]
@smarterclayton bump

**Comment by bparees** [comment]
self-approving, just changes tagging on e2e tests.

**Comment by bparees** [comment]
@stevekuznetsov this PR seems stuck...

**Pull request merged**